ikasaneip/rest/rest-module/src/main/java/org/ikasan/rest/module/ModuleControlApplication.java

```diff
@@ -291,5 +292,43 @@ else if (action.equalsIgnoreCase("stop"))
         return new ResponseEntity("Context Listeners state changed successfully!", HttpStatus.OK);
     }
 
+    @RequestMapping(method = RequestMethod.PUT,
+        value = "/activator/{moduleName}/{action}")
+    @PreAuthorize("hasAnyAuthority('ALL','WebServiceAdmin')")
+    public ResponseEntity activator(@PathVariable("moduleName") String moduleName,
+                                    @PathVariable("action") String action)
+    {
+        try
+        {
+            Module<Flow> module = moduleService.getModule(moduleName);
+
+            if (action.equalsIgnoreCase("activate"))
+            {
+                this.moduleActivator.activate(module);
+            }
+            else if (action.equalsIgnoreCase("deactivate"))
+            {
+                this.moduleActivator.deactivate(module);
+            }
+            else
+            {
+                return new ResponseEntity("Unknown module activation action [" + action + "].", HttpStatus.FORBIDDEN);
+            }
+        }
+        catch (Exception e)
+        {
+            return new ResponseEntity(e.getMessage(), HttpStatus.FORBIDDEN);
+        }
+        return new ResponseEntity(String.format("Module action[%s] successfully applied!", action), HttpStatus.OK);
+    }
+
+    @RequestMapping(method = RequestMethod.GET,
+        value = "/isActivated/{moduleName}")
```

Comment:
why do you need to provide module name?
we have never had usecase where we had more than 1 module inside ikasan app  shouldnt call just be is Activated?